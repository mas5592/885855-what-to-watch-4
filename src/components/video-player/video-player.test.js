import React from 'react';
import rerender from 'react-test-renderer';

import VideoPlayer from './video-player.jsx';

const film = {
  id: 1,
  title: `Aviator`,
  img: `img/aviator.jpg`,
  release: 2004,
  genre: `Dramas`,
  poster: `https://avatars.mds.yandex.net/get-kinopoisk-image/1599028/5d4dbb46-36e2-4892-a71d-fded5d0fe86d/x1000`,
  coverBackground: `img/aviator.jpg`,
  rating: {
    score: 8.9,
    level: `Very good`,
    count: 240
  },
  description: `In the 1930s, the Grand Budapest Hotel is a popular European ski resort, presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, becomes Gustave's friend and protege. Gustave prides himself on providing first-class service to the hotel's guests, including satisfying the sexual needs of the many elderly women who stay there. When one of Gustave's lovers dies mysteriously, Gustave finds himself the recipient of a priceless painting and the chief suspect in her murder.`,
  team: {
    director: `Wes Andreson`,
    starring: `Bill Murray, Edward Norton, Jude Law, Willem Dafoe and other`
  },
  preview: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`
};

it(`Should VideoPlayer render correctly`, () => {
  const tree = rerender.create(
      <VideoPlayer
        isMuted={true}
        isPlaying={true}
        poster={film.poster}
        src={film.preview}
      />,
      {
        createNodeMock: () => {
          return {};
        }
      }).toJSON();

  expect(tree).toMatchSnapshot();
});
