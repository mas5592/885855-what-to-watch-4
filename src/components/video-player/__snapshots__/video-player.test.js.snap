// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer Should render VideoPlayer component 1`] = `
<video
  className="player__video"
/>
`;
